requirements.txt

```diff
@@ -2,7 +2,7 @@
 git+https://github.com/tdryer/hangups.git
 appdirs
 asyncio
-aiohttp
+aiohttp>=1.3,<2 # telepot library also needs this version
```

Comment:
there is an obvious issue with here as this is mutually exclusive with hangups `setup.py` which is [`<1.3`](https://github.com/tdryer/hangups/blob/master/setup.py#L69) - currently testing a hangups build on aiohttp-1.3.5